docs/specification/current.md

```diff
@@ -1095,6 +1095,76 @@ As each hook progresses through a process of being defined, tested, implemented,
+Coordinating `order-select` and `order-sign` hooks involves sending configuration information to the CDS Service which prompts the service to cache data from the `order-select` request and use that data prior to returning cards from an `order-signed` request to determine if the cards would appear a duplicates to the clinician. The figure below depicts the summary of operations that result in coordination between the `order-select` and `order-sign` hooks.
+
+![cds service ](../images/cds_service.png)
+
+**CDS Discovery Response**
+Prior to a hook trigger and subsequent processes, the EHR MUST initiate a CDS Discovery. CDS Discovery is to identify CDS services and obtain associated prefetch templates. A prefetch template is a dictionary of read and search requests for needed resources of a particular service.
+In addition to the attributes specified in the [Discovery](#discovery) section of this page, a new `extension` resource will be required.
+
+Field | Optionality | Type | Description
+----- | ----- | ----- | ---------
+`extension` | REQUIRED | *object* | 
+ 
+ 
+```
+ {
+  "services": [
+    {
+      "hook": "order-select",
+      "title": "Static CDS Service Example",
+      "extension": {
+        "configuration-items": [
+          {
+            "code": <an identifier for the configuration option>,
+            "type": <data type of the configuration option, e.g. boolean>,
+            "name": <human readable name of the configuration option>,
+            "description": <human readable description of the configuration option>
+          },
+
+          ... other configuration options ...
+        ]
+    },
+    {
+      "hook": "order-sign",
+      "title": "Static CDS Service Example",
+      "extension": {
+        "configuration-items": [
+          {
+            "code": <an identifier for the configuration option>,
+            "type": <data type of the configuration option, e.g. boolean>,
+            "name": <human readable name of the configuration option>,
+            "description": <human readable description of the configuration option>
+          },
+
+          ... other configuration options ...
+        ]
+      }, ...
+
+snipped for brevity
+```
+
+##### Extension
+
+###### Configuration Items
+
+The CDS Service MUST process configuration options sent as within an extension resource with the CDS Hooks request in the field `configuration-items`. There are two standart configuration options:  `cache-for-order-sign-filtering` (only used in `order-select`), `filter-out-repeated-alerts` (only used in `order-sign`). All of these configuration options accept Boolean values and are required. The `cache-for-order-sign-filtering` option is used to cache the medication resource during `order-select` and is used as a reference during `order-sign` if `filter-out-repeated-alerts` is set to TRUE. The `filter-out-repeated-alerts` option is used to hide card results if they were shown in the `order-select` results.
```

Comment:
The 1.0 spec [states](https://cds-hooks.hl7.org/1.0/#extensions):
> The specification is not prescriptive about support for extensions. However, to support extensions, the specification reserves the name extension and will never define an element with that name

The spec allows for extensions but does not define them. 
 Based on that, any kind of functionality like this would need to be directly supported elements that can be applied across any number of hooks.